.travis.yml

```
@@ -10,16 +10,14 @@ before_install:
   - bundle install
 
   
-# before_script:
-#   - cp config/database.travis.yml config/database.yml
-#   - psql -c "CREATE USER \"wbooks-api\" WITH PASSWORD 'wbooks-api';" -U postgres
-#   - psql -c "DROP DATABASE IF EXISTS \"wbooks_api_test\";" -U postgres
-#   - psql -c "CREATE DATABASE \"wbooks_api_test\" ENCODING 'utf8';" -U postgres
-#   - bundle exec rake db:migrate
+before_script:
+  - cp config/database.travis.yml config/database.yml
+  - psql -c "CREATE USER \"wbooks-api\" WITH PASSWORD 'wbooks-api';" -U postgres
+  - psql -c "DROP DATABASE IF EXISTS \"wbooks_api_test\";" -U postgres
+  - psql -c "CREATE DATABASE \"wbooks_api_test\" ENCODING 'utf8';" -U postgres
+  - bundle exec rake db:migrate
   
 
 script:
-  - bundle exec rubocop app
-
-cache:
-  bundler: true
\ No newline at end of file
+  - bundle exec rubocop app spec/controller
```

Change the command to include all spec folder 
```suggestion
  - bundle exec rubocop app spec
```